### PR TITLE
expanding on definition of valid URI

### PR DESCRIPTION
**Pull request by teehill**

### CODE REVIEW

File: versions/1.0/index.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Servers **MUST** send all JSON data in response documents with the header
 
 Anywhere a **URI** is specified, it must adhere to the following rules:
 
-* **MUST** be a valid URI.
+* **MUST** be a valid URI that is active and responds to requests.
```
Comment (ardiesaeidi) [inline versions/1.0/index.md:29]: How about `Must be a valid URI that is active and returns a resource`? Also should we add language around not having empty paths within the URI that are solely used for grouping?

Comment (ardiesaeidi) [reply versions/1.0/index.md:29]: @teehill actually can we revert back to how you had it originally? I think your wording is more generic and covers situations where an endpoint does not return a resource. (ie POST only calls)

Comment (teehill) [reply versions/1.0/index.md:29]: done

**Comment by ardiesaeidi** [comment]
This change was included in https://github.com/VerizonDigital/hyperion/pull/17

**Pull request closed without merging**